msteams-platform/concepts/tdp-configuration.md

```diff
@@ -0,0 +1,108 @@
+
+##### Connector
+
+This section allows you to add a connector to your app. If you already have registered an Office 365 connector, choose *Set up* and enter the name and ID of the connector. If you want a new connector click *Register* to be taken to the Connector Developer Dashboard in your browser.
+
+> [!NOTE]
+> App customization enables admins to change the look-and-feel of the apps loaded through bots, messaging extensions, tabs, and connectors. For example, if the Teams admin customizes the name of an app from *Contoso* to *Contoso Agent*, then the app will appear with the new name *Contoso Agent* to users. However, while adding a connector to a chat, in the list the connectors will still show the name of the app as *Contoso*.
+
+##### Messaging Extensions
+
+[Messaging extensions](~/messaging-extensions/what-are-messaging-extensions.md) are a powerful way for users to engage with your app within Microsoft Teams. Users can query for information from your service and post that information in the form of cards, right into the channel or chat conversation.
+
+Messaging extensions are powered by Bot Framework bots, so they require a configured bot to operate. If you have the name and Bot Framework ID of the bot you would like to power the messaging extension, enter it. Otherwise, click *Register* to create one and enter the information afterward. Select whether the configuration of a messaging extension can be updated by the user.
+
+Once you have the underlying bot configured, define the commands and parameters which the messaging extension can accept.
+
+Each command requires a title and an ID. The command can optionally contain a description for the user. Each command can support up to five parameters, each of which requires:
+
+* The name of the parameter as it appears in the Teams client and is included in the user request
+* A user-friendly title
+* An optional description
+
+> [!NOTE]
+> To create messaging extension using app studio, see [create messaging extension using app studio](~/resources/create-messaging-extension-using-appstudio.md).
+
+##### Meeting extension
+
+//TODO: Rajesh R.
+
+##### Scene
+
+Scenes in Together Mode is an artifact created by the scene developer using the Microsoft Scene studio that brings people together along with their video stream in a creative setting as conceived by the scene creator. In a conceived scene setting, participants have designated seats with video streams rendered in those seats. For more information, see [Teams Together Mode](~/apps-in-teams-meetings/teams-together-mode.md).
+
+#### Permissions
+
+You can enrich your Teams app with native device capabilities, such as camera, microphone, and location.
+
+#### Languages
+
+Set up or change the languages that your app supports.
+
+#### Single Sign-On
+
+Configure your app to authenticate users with single sign-on (SSO).
+
+#### Domains
+//TODO
```

Comment:
@heath-hamilton You can base this content on:
https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#validdomains